catalog/ui/catalog-ui-search/src/main/webapp/component/metacard-actions/metacard-actions.view.js

```diff
@@ -44,6 +43,18 @@ define([
                 this.setDefaultModel();
             }
         },
+        serializeData: function() {
+            return {
+                exportActions: _.sortBy(this.model.getExportActions().map(action => ({
+                    url: action.get('url'),
+                    title: action.get('title').replace('Export as', '').replace('Export','')
```

Comment:
This replacement should only remove `"Export"` or `"Export as"` from the beginning of titles and should also ensure that whitespace after the `"Export"` or `"Export as"` is removed so as to not leave leading white space on the resulting text. On top of that, if `"as"` begins an export format, e.g., "Export asterisks file", then the `"as"` will be dropped accidentally. I suggest using this instead:
```
title: action.get('title').replace(/^Export( as)?\s+\b/, '')
```